### PR TITLE
added TxtCsvHelper

### PR DESCRIPTION
**Pull request by camdrudge**
added new nuget package

### CODE REVIEW

File: NCsvPerf.Test/NCsvPerf.Test.csproj [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.3" />
+    <PackageReference Include="TxtCsvHelper" Version="1.2.8" />
```
Comment (joelverhagen) [inline NCsvPerf.Test/NCsvPerf.Test.csproj]: This is not needed since this reference will come in transitively from NCsvPerf.csproj.

**Comment by camdrudge** [comment]
I'm sorry, how do I include that then?

Sorry to be such a pain, like I said I'm fully self taught and I have
little experience with Github and unit testing

**Comment by joelverhagen** [comment]
> I'm sorry, how do I include that then? Sorry to be such a pain, like I said I'm fully self taught and I have little experience with Github and unit testing

No problem :). Because the NCsvPerf.Test.csproj project file references the NCsvPerf.csproj project with a project reference, all of the package references in the latter will automatically flow in to the test project. This is why you don't see any other <PackageReference> lines in NCsvPerf.Test.csproj for the other CSV parsing libraries that are being tested. You can simply delete that line that I commented on and run `dotnet build` again to see that it still works.

**Comment by camdrudge** [comment]
Okay, it should be fixed now!

**Comment by joelverhagen** [comment]
Thanks Cameron!

**Pull request merged**